app/views/projects/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Project Name: <%= @project.name %></h1>
+<h3>Material: </h3><%= @project.material %>
+<h3>Challenge Theme: </h3><%= @project.challenge.theme %>
+<h3>Number of Contestants: </h3><%= @project.contestants.count %>
+
```

Comment:
All rendered erb tags need to be enclosed in HTML.

```erb
<h1>Project Name: <%= @project.name %></h1>
<h3>Material: <%= @project.material %></h3>
<h3>Challenge Theme: <%= @project.challenge.theme %></h3>
<h3>Number of Contestants: <%= @project.contestants.count %></h3>
```

You also mentioned this in your reflection, but you don't want to do a `count` in the view. This is database logic and should go in the model.